.gitlab/ci-images.yml

```diff
@@ -115,3 +115,83 @@ Ubuntu Buster:
+        IMAGE_TAG: "datadog/dd-trace-ci:php-8.1_windows"
+        DOCKERFILE: "Dockerfile"
+        ARGS: "--build-arg phpVersion=8.1.29 --build-arg vsVersion=vs16 --build-arg phpTarGzUrl=https://www.php.net/distributions/php-8.1.29.tar.gz --build-arg phpSha256Hash=8b2609bf1d3173aa38269a9af21532c65f730aadd3051f9aae011eea9e246de5"
+      - BUILD_NAME: "php-8.0"
+        IMAGE_TAG: "datadog/dd-trace-ci:php-8.0_windows"
+        DOCKERFILE: "Dockerfile"
+        ARGS: "--build-arg phpVersion=8.0.28 --build-arg vsVersion=vs16 --build-arg phpTarGzUrl=https://www.php.net/distributions/php-8.0.28.tar.gz --build-arg phpSha256Hash=7432184eae01e4e8e39f03f80e8ec0ca2c8bfebc56e9a7b983541ca8805df22f"
+      - BUILD_NAME: "php-7.4"
+        IMAGE_TAG: "datadog/dd-trace-ci:php-7.4_windows"
+        DOCKERFILE: "Dockerfile"
+        ARGS: "--build-arg phpVersion=7.4.33 --build-arg vsVersion=vc15 --build-arg phpTarGzUrl=https://www.php.net/distributions/php-7.4.33.tar.gz --build-arg phpSha256Hash=5a2337996f07c8a097e03d46263b5c98d2c8e355227756351421003bea8f463e"
+      - BUILD_NAME: "php-7.3"
+        IMAGE_TAG: "datadog/dd-trace-ci:php-7.3_windows"
+        DOCKERFILE: "Dockerfile"
+        ARGS: "--build-arg phpVersion=7.3.33 --build-arg vsVersion=vc15 --build-arg phpTarGzUrl=https://www.php.net/distributions/php-7.3.33.tar.gz --build-arg phpSha256Hash=9a369c32c6f52036b0a890f290327f148a1904ee66aa56e2c9a7546da6525ec8"
+      - BUILD_NAME: "php-7.2"
+        IMAGE_TAG: "datadog/dd-trace-ci:php-7.2_windows"
+        DOCKERFILE: "Dockerfile"
+        ARGS: "--build-arg phpVersion=7.2.34 --build-arg vsVersion=vc15 --build-arg phpTarGzUrl=https://www.php.net/distributions/php-7.2.34.tar.gz --build-arg phpSha256Hash=8b2777c741e83f188d3ca6d8e98ece7264acafee86787298fae57e05d0dddc78"
+      - BUILD_NAME: "php-7.1"
+        IMAGE_TAG: "datadog/dd-trace-ci:php-7.1_windows"
+        DOCKERFILE: "Dockerfile"
+        ARGS: "--build-arg phpVersion=7.1.33 --build-arg vsVersion=vc14 --build-arg phpTarGzUrl=https://www.php.net/distributions/php-7.1.33.tar.gz --build-arg phpSha256Hash=0055f368ffefe51d5a4483755bd17475e88e74302c08b727952831c5b2682ea2"
+      - BUILD_NAME: "php-7.0"
+        IMAGE_TAG: "datadog/dd-trace-ci:php-7.0_windows"
+        DOCKERFILE: "Dockerfile"
+        ARGS: "--build-arg phpVersion=7.0.33 --build-arg vsVersion=vc14 --build-arg phpTarGzUrl=https://www.php.net/distributions/php-7.0.33.tar.gz --build-arg phpSha256Hash=d71a6ecb6b13dc53fed7532a7f8f949c4044806f067502f8fb6f9facbb40452a"
+  hooks:
+    pre_get_sources_script:
+      - git config --system core.longpaths true
+      - echo "{}" > C:\\Windows\\system32\\config\\systemprofile\\.docker\\config.json
+      - type C:\\Windows\\system32\\config\\systemprofile\\.docker\\config.json
+  script:
+    - |
+      cd dockerfiles/ci/windows
+      docker login -u "$CI_REGISTRY_USER" -p "$CI_REGISTRY_TOKEN" $CI_REGISTRY
+      docker build --platform windows/amd64 -t $IMAGE_TAG -f $DOCKERFILE $ARGS .
```

Comment:
Was is not possible to use the docker-compose.yml? I'd lover to avoid mutliple sources of truth here, if possible.

Reply:
nope, no `docker buildx`, no `docker compose` on the windows machines 😢

Reply:
Decided to drop Windows builds as long as there is no `docker compose` support in Windows runners